content/en/docs/gke/deploy/management-setup.md

```diff
@@ -9,118 +9,220 @@ This guide describes how to setup a management cluster which you will use to dep
 
-1. Fetch the management blueprint
+    ```bash
+    kpt pkg get https://github.com/kubeflow/gcp-blueprints.git/management@v1.2.0 ./
+    ```
 
-   ```
-   kpt pkg get https://github.com/kubeflow/gcp-blueprints.git/management@v1.1.0 ./
-   ```
+1.  Fetch the upstream management package
 
-1. Fetch the upstream manifests
+    ```bash
+    cd ./management
+    make get-pkg
+    ```
 
-   ```
-   cd ./management
-   make get-pkg
-   ```
+    Note, all the instructions below assume your current working directory is `./management`.
 
-1. Open up the **Makefile** at `./management/Makefile` and edit the `set-values` rule to set values for the name, project, and location of your management; when you are done the section should look like
+1.  Use kpt to set values for the name, project, and location of your management cluster:
 
-   ```  
-    set-values: 
-    kpt cfg set ./instance name NAME
-    kpt cfg set ./instance location LOCATION
-    kpt cfg set ./instance gcloud.core.project PROJECT
-  
-    kpt cfg set ./upstream/management name NAME
-    kpt cfg set ./upstream/management location LOCATION
-    kpt cfg set ./upstream/management gcloud.core.project PROJECT
+    ```bash
+    kpt cfg set -R . name ${NAME}
+    kpt cfg set -R . gcloud.core.project ${PROJECT}
+    kpt cfg set -R . location ${LOCATION}
+    ```
 
-   ```
+    For the values you need to set for management cluster:
 
-   * Where **NAME**, **LOCATION**, **PROJECT** should be the actual values for your deployment
+    - NAME is the cluster name of your management cluster. Management cluster
+      should be a different cluster from your Kubeflow cluster.
 
-1. Set the values
+      Note, NAME should
 
-   ```
-   make set-values
-   ```
+      - start with a lowercase letter
+      - only contain lowercase letters, numbers and `-`
+      - end with a number or a letter
+      - contain no more than 18 characters
 
-1. Hydrate and apply the manifests to create the cluster
+    - LOCATION is the management cluster's location, you can choose between regional or zonal.
+    - PROJECT is the Google Cloud project where you will create this management cluster.
 
-   ```
-   make apply
-   ```
+    Running `kpt cfg set` stores values you set in `./instance/Kptfile` and
+    `./upstream/management/Kptfile`. Commit the changes to source control to
+    preserve your configuration.
 
-1. Create a kubeconfig context for the cluster
+    We have two packages with setters: `./instance` and `./upstream/management`.
+    The `-R` flag means `--recurse-subpackages`. It sets values recursively in all the
+    nested subpackages in current directory `.` in one command.
 
-   ```
-   make create-ctxt
-   ```
+    You can learn more about `kpt cfg set` in [kpt documentation](https://googlecontainertools.github.io/kpt/reference/cfg/set/), or by running `kpt cfg set --help`.
 
-1. Install CNRM
+    Note, you can find out which setters exist in a package and which values were previously set by:
```

Comment:
> which values were previously set

Feels like it should be more about the the current state instead of the past?

Reply:
Good point!